anaconda_anon_usage/patch.py

```diff
@@ -13,13 +13,15 @@
 
 def _new_user_agent(ctx):
     result = ctx._old_user_agent
-    if context.anaconda_anon_usage:
-        prefix = Context.checked_prefix or context.target_prefix or sys.prefix
-        token = token_string(prefix)
+    prefix = (
+        getattr(Context, "checked_prefix", None) or context.target_prefix or sys.prefix
```

Comment:
Added the `getattr` check here to handle the case in which the `checked_prefix` monkeypatch hasn't been created (e.g. via a messed up patch). I don't think we should automatically ignore that error since the target prefix or sys prefix should suffice as a fallback. We also still want to get the information that aau is installed on those systems in this edge case.